test/e2e/suite/conformance/certificates/suite.go

```diff
@@ -84,6 +84,8 @@ func (s *Suite) complete(f *framework.Framework) {
 			s.DomainSuffix = f.Config.Addons.IngressController.Domain
 		case "Gateway":
 			s.DomainSuffix = f.Config.Addons.Gateway.Domain
+		default:
+			s.DomainSuffix = "example.com"
```

Comment:
Was this causing test failures? Under what  circumstances is DomainSuffix empty?

Reply:
This did not cause failures, but the value was empty and thus certificates were created with eg. `dqfdsfs.` as domain (instead of `dqfdsfs.example.com`) So I think it just worked by incident. This bug was introduced in #4276, before that the default was just the IngressController domain name.